frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -241,15 +243,15 @@ class Slider extends React.PureComponent<Props, State> {
       return (
         <StyledThumb
           {...passThrough}
-          disabled={props.$disabled}
+          disabled={props.$disabled ? props.$disabled : false}
```

Comment:
nit: I think you can simplify this to `props.$disabled || false` or just `props.$disabled === true`

Reply:
Sure 👍 It's changed 👍